plugin/environment.world

```diff
@@ -59,16 +64,16 @@
         <uri>model://teensize_goal</uri>
       </include>
       <pose>4.525 0 0 0 0 0</pose>
-    </model> 
+    </model>
 
     <!-- The ball -->
     <model name="nubots_ball">
       <include>
         <uri>model://teensize_ball</uri>
       </include>
       <pose>0 0 0.15 0 0 0</pose>
-      <plugin name="nubots_ball" filename="libnubotsball_plugin.so"/>
-    </model> 
+      <plugin name="nubots_ball" filename="libball_plugin.so"/>
```

Comment:
We may want to make these names more unique. There are a lot of other plugins in the plugins folder